recipes/belt/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "BELT" %}
+{% set version = "BELTv1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qianglbl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: cbfe5081595ba91bb88e4cb9deec001f60c73571a8cc10788ac60d789089f843
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  script: 
+    - ln -sf $BUILD_PREFIX/bin/$CONDA_TOOLCHAIN_HOST-gfortran $BUILD_PREFIX/bin/gfortran
+    - cd src
+    - make -f Makefile
```

Looks like this also might work
```suggestion
    - cd src
    - make FORTRAN=$FORTRAN
```

---

`make FORTRAN=$FORTRAN` did not work, but `make FORTRAN=$GFORTRAN` did work.

---

I can find no solution other than the soft link that allows the code to compile on the conda-forge runners.

---

[Looks like you need](https://github.com/qianglbl/BELT/blob/main/src/Makefile#L28):

```
make CC=$FORTRAN LINK=$FORTRAN
```

---

@chrisburr That works!  I have committed an update to the `make` command as suggested.